src/cmd/serial.test.js

```diff
@@ -53,5 +61,21 @@ describe('Serial Command', () => {
 			});
 		});
 	});
+
+	describe('serialWifiConfig', () => {
+		it('can reject with timeout after 5000ms if not getting any serial data', () => {
+			clock = sinon.useFakeTimers();
```

Comment:
lol i always forget about `useFakeTimers()` 🤦‍♂ 

there are probably a few tests bouncing around that could benefit from that.